templates/common/gcp/files/etc-networkmanager-conf.d-hostname.yaml

```diff
@@ -0,0 +1,10 @@
+mode: 0644
+path: "/etc/NetworkManager/conf.d/hostname.conf"
+contents:
+  inline: |
+    # On GCP the hostname is likely to be longer than 63 chars.
+    # /etc/networkmanager/dispatcher.d/90-long-hostname.sh will
+    # the hostname. The following configuration allows 90-long-hostname.sh
+    # to manage setting transient hostname instead of NetworkManager itself.
```

Comment:
nit: drop a link to the NM docs for this setting?  https://developer.gnome.org/NetworkManager/stable/NetworkManager.conf.html

Reply:
Ha. I wouldn't call this a nit -- the doc llink helps explain this. Also, I added the BZ ref.